docs/Frequently Asked Questions/Multiple Processes/index.md

```diff
@@ -1,20 +1,20 @@
+The router service is designed to live outside the normal lifecycle of the app integrating the SDL framework. The different process allows a level of security to cut off access to the hosting application's data because Android allocates a different memory space for the router service process to run. It also allows the router service to not interfere with the hosting application's runtime; this means if the router service unexpectedly stops or crashes, it will not take down the hosting app. This relationship also works in the opposite direction, which is important to maintain a good user experience when apps are connected through a router service.
 
-## Content providers and multiple process issues
+## Content providers and multiple processes
 
-Android content providers have a unique lifecycle that does not work in the expected flow. Content providers are actually started before the `Application` class and following Activities, Services, etc. Some libraries use this to know when their code/module can initialize and always be ready for the entire lifecycle of the application. This is found with many Google libraries (Firebase, Jetpack, etc).
+Android content providers have a unique lifecycle that does not work in the expected flow. Content providers are actually started before the `Application` class and following `Activities`, `Services`, etc. Some libraries use this to know when their code/module can initialize and always be ready for the entire lifecycle of the application. This is found with many Google libraries (Firebase, Jetpack, etc).
 
-The issue is that, by default, content providers are only attached to the main process. This process is the same as the application package unless otherwise specified. This means, when the main process starts the content provider will be started, but if a different process is started the content provider will not be started. So if the app has its first start from a component that is designed to run in a different process, the content provider won't be ready by the time those components start up; this includes the application instance for that process. 
+The issue is that, by default, content providers are only attached to and initialized for the main process. This means, when the main process starts the content provider will be started, but if a different process other than the main processes is started the content provider will not be started. So if the app has its first start from a component that is designed to run in a different process, the content provider won't be ready by the time those components start up; this includes the `Application` instance for that process. 
```

Comment:
Should this be `...other than the main process is started...` ?  You don't mention "main processes" anywhere but here.